index.js

```diff
@@ -18,6 +18,7 @@ const state = {
   selectedOrgGags: _selectedOrgGags,
   selectedOrgGagCounts: calcSelectedGagTrackCounts(_selectedOrgGags),
   isLured: savedState?.isLured || false,
+  toggle: savedState?.toggle || false,
```

Comment:
Could this toggle be named something more descriptive, such as `hideLvl13UpCogs`?

Reply:
Yes, I'll do that.